### PR TITLE
Agregando testng.xml

### PR DESCRIPTION
**Pull request by ycedeno3**

### CODE REVIEW

File: src/main/java/test/dataProviderEmailTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package test;
+
+import Page.DataProvidersClass;
+import Page.Prueba_Netflix;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+import utilities.Utilities;
+
+public class dataProviderEmailTest extends Prueba_Netflix {
```
Comment (aimiebell) [inline src/main/java/test/dataProviderEmailTest.java:9]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: src/main/java/test/fakeEmailTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package test;
+
+import Page.Prueba_Netflix;
+import Page.fakerClass;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utilities.Utilities;
+
+public class fakeEmailTest extends Prueba_Netflix {
```
Comment (aimiebell) [inline src/main/java/test/fakeEmailTest.java:10]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: src/main/java/test/inicioSessionPageTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+import Page.Prueba_Netflix;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class inicioSessionPageTest extends Prueba_Netflix {
```
Comment (aimiebell) [inline src/main/java/test/inicioSessionPageTest.java:11]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: src/main/java/test/loginToNetflixErrorTest.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package test;
+
+import Page.Prueba_Netflix;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utilities.Utilities;
+
+public class loginToNetflixErrorTest extends Prueba_Netflix {
```
Comment (aimiebell) [inline src/main/java/test/loginToNetflixErrorTest.java:9]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: src/main/java/test/printTagsTest.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+import Page.Prueba_Netflix;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class printTagsTest extends Prueba_Netflix {
```
Comment (aimiebell) [inline src/main/java/test/printTagsTest.java:11]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: src/main/java/test/validarTituloTest.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package test;
+
+import Page.Prueba_Netflix;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class validarTituloTest extends Prueba_Netflix {
```
Comment (aimiebell) [inline src/main/java/test/validarTituloTest.java:7]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: src/main/java/test/inicioSessionPageTest.java [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    @Test
+    public void inicioSessionPage() throws InterruptedException {
+        driver.findElement(By.xpath("//a[contains(text(),'Iniciar sesión')]")).click();
+        String tituloEsperado;
+        tituloEsperado = "Netflix";
+        String tituloAntiguo = "Netflix Chile: Ve series online, ve películas online";
+        String tituloActual = "";
+        tituloActual = driver.getTitle();
+        System.out.println(tituloActual);
+        Assert.assertEquals(driver.getTitle(),
```
Comment (seleniumcurso) [inline src/main/java/test/inicioSessionPageTest.java:22]: esto podria ser simplificado

**Comment by seleniumcurso** [comment]
Faltan resolver algunos puntos. 
No era necesario crear una clase por cada test, sino que bastaba con realizar una unica clase de test y ahi agregar todos los tests. 
Se pueden simplificar algunos metodos
En resumen, si bien hay cosas a mejorar, se muestra dominio de la herramienta selenium y se cumple gran parte de lo solicitado. El examen esta aprobado!

**Pull request merged**